### DISCUSSION TITLE
Docker Extension VSC - "failed fetching" error when executing "Run and debug" on a container

### DISCUSSION THREAD

**Discussion by adijian** [microsoft/vscode-docker#4370] [Q&A]
On version 1.29.2 there is an issue running and debugging a containered .NET app.
A pop up message saying "failed fetching" appears with no further information.

Encountered when following this guide: https://code.visualstudio.com/docs/containers/quickstart-aspnet-core

No issue on version 1.23.0.

**Comment by bwateratmsft** [comment]
Can you share a screenshot?

**Reply by adijian** [reply]
![image](https://github.com/user-attachments/assets/f896d1d4-af08-452f-9815-fce92cc48e27)

**Reply by bwateratmsft** [reply]
Is this when you do F5?

**Reply by bwateratmsft** [reply]
Also, is there any more info available in the Output tab > Docker output log?

**Reply by adijian** [reply]
yes it happens when I F5

**Reply by adijian** [reply]
No, nothing, it initally pops up acquiring .net debugger, then it says fetch failed 
![image](https://github.com/user-attachments/assets/56f2c873-417d-4a91-adb6-49ec612a696b)

**Reply by bwateratmsft** [reply]
Sounds like it's failing to get the VSDBG download script. Are you behind a restrictive firewall or proxy?

**Reply by adijian** [reply]
nope, happens on my personal pc out of work as well!

**Reply by bwateratmsft** [reply]
If 1.23.0 is working this is probably a consequence of the change from `node-fetch` to plain old `fetch`. I'll look at adding some additional logging here so we can better figure out what is going on.

**Reply by bwateratmsft** [reply]
I created https://github.com/microsoft/vscode-docker/issues/4378 to track that.
